app/src/main/res/drawable/ic_tutorial_img_2.xml

```diff
@@ -0,0 +1,11 @@
+    <path android:fillColor="#878787" android:pathData="M148.15,40.65A20.25,20.25 0,1 0,128 20.37,20.19 20.19,0 0,0 148.15,40.65Z"/>
+    <path android:fillColor="#878787" android:pathData="M222,87c0.86,-1.56 1.65,-3 2.38,-4.37a1,1 0,0 1,1.33 -0.57c2,0.56 4.08,1 6.38,1.58l-1.45,-5.38 7.4,-1.34L232.8,73 238,69l-7.4,-1.29L232,62.37c-2,0.5 -4,0.86 -5.81,1.47a1.53,1.53 0,0 1,-2.22 -1c-0.6,-1.26 -1.35,-2.44 -2.13,-3.83 -1.38,1.28 -2.71,2.45 -4,3.7a1.21,1.21 0,0 1,-1.67 0.24c-1.71,-0.92 -3.47,-1.78 -5.38,-2.75 -0.19,1.52 -0.41,2.92 -0.52,4.32 -0.07,0.85 -0.36,1.16 -1.25,1.14 -2.11,-0.07 -4.23,0 -6.45,0L206,70.34l-6.6,2.76L206,75.83l-3.39,4.73h7.57l0.78,5.31c1.87,-1 3.58,-1.79 5.25,-2.71a1.3,1.3 0,0 1,1.79 0.21C219.3,84.58 220.63,85.73 222,87Z"/>
+    <path android:fillColor="#878787" android:pathData="M196.21,41.55c-0.25,0.48 -0.46,0.86 -0.65,1.25L190,54.33c-1.95,4 -3.91,8.05 -5.85,12.07 -0.21,0.44 -0.63,0.9 0.14,1.29 4.34,2.17 8.66,4.41 13,6.55 1.76,0.87 3.61,1.57 5.5,2.38l-4.29,6c1.07,-0.05 1.78,-0.1 2.49,-0.12s1.37,0 2.16,0A3.58,3.58 0,0 1,198.05 84q-8,-3.85 -16,-7.74c-2.92,-1.43 -3.53,-3.19 -2.11,-6.12q4.68,-9.64 9.37,-19.28c2.34,-4.82 4.66,-9.64 7,-14.44 1,-2.07 3,-2.9 4.85,-2q8.7,4.14 17.35,8.42a3.32,3.32 0,0 1,1.74 4,14.08 14.08,0 0,1 -1,2.31c-1.7,3.52 -3.41,7 -5.15,10.62l-1.82,-0.93 3.69,-7.68ZM191.93,73A3,3 0,0 0,189.09 76a2.67,2.67 0,0 0,2.62 2.81,3 3,0 0,0 2.78,-3A2.67,2.67 0,0 0,191.93 73ZM205.21,42.14c1.58,0.83 3.21,1.57 4.81,2.36a1.19,1.19 0,0 0,1.8 -0.55c0.41,-0.81 0,-1.37 -0.73,-1.73 -1.61,-0.79 -3.2,-1.59 -4.82,-2.34a1.32,1.32 0,0 0,-2 1.37A3.73,3.73 0,0 0,205.21 42.18Z"/>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/8aa95bc22269ef11a970c62e715e84ca8f84ab7e/app/src/main/res/drawable/ic_tutorial_img_2.xml#L10"></span>Very long vector path (800 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.</td>
    </tr>
  </tbody>
</table>